DiverseSelector/test/test_metric.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# The DiverseSelector library provides a set of tools to select molecule
+# subset with maximum molecular diversity.
+#
+# Copyright (C) 2022 The QC-Devs Community
+#
+# This file is part of DiverseSelector.
+#
+# DiverseSelector is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# DiverseSelector is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+
+"""Metric calculation test module."""
```

Comment:
This is not a testing yet. Some resources, 
1. https://docs.pytest.org/en/7.1.x/
2. https://github.com/theochem/procrustes/tree/master/procrustes/test